packages/nodejs/src/span.ts

```diff
@@ -125,6 +126,7 @@ export class BaseSpan implements Span {
         >
   ): this {
     if (!key || !data) return this
+    if (key == "params" && !BaseClient.config.data.sendParams) return this
```

Comment:
Could this ever be `undefined`? If a user hasn't initialized the config yet for example.

Reply:
No user should call this function without a configuration initialized. Spans are born from the Tracer object, which needs an initialized config. It'll fail before getting to this condition.

Reply:
Look like it would break before it arrives here, with the integrations calling `tracer()` on an empty/undefined AppSignal client trying to start a span. When we rely on the globally stored client more we could return a "noop" client object if needed. Let's revisit it later if needed.